### ISSUE TITLE
Breaking change in wagtail 2.4

### ISSUE THREAD

**Issue by ChickenF622** [bashu/wagtail-embedvideos#63]
The following error is thrown when trying to choose an embedded video in an inline video chooser panel

```
Internal Server Error: /cms/embed_videos/chooser/
Traceback (most recent call last):
  File "/home/chickenf622/vscodeprojects/mywebsite/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/chickenf622/vscodeprojects/mywebsite/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/chickenf622/vscodeprojects/mywebsite/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/chickenf622/vscodeprojects/mywebsite/venv/lib/python3.6/site-packages/django/views/decorators/cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/home/chickenf622/vscodeprojects/mywebsite/venv/lib/python3.6/site-packages/wagtail/admin/urls/__init__.py", line 102, in wrapper
    return view_func(request, *args, **kwargs)
  File "/home/chickenf622/vscodeprojects/mywebsite/venv/lib/python3.6/site-packages/wagtail/admin/decorators.py", line 34, in decorated_view
    return view_func(request, *args, **kwargs)
  File "/home/chickenf622/vscodeprojects/mywebsite/venv/lib/python3.6/site-packages/wagtail_embed_videos/views/chooser.py", line 82, in chooser
    'popular_tags': popular_tags_for_model(EmbedVideo),
  File "/home/chickenf622/vscodeprojects/mywebsite/venv/lib/python3.6/site-packages/wagtail/admin/modal_workflow.py", line 11, in render_modal_workflow
    raise TypeError("Passing a js_template argument to render_modal_workflow is no longer supported")
TypeError: Passing a js_template argument to render_modal_workflow is no longer supported
```

**Comment by ChickenF622** [comment]
Fix for this issue can be found in #64

**Comment by DiogoMarques29** [comment]
With the introduction of Embedded Content in wagtail 1.12 https://docs.wagtail.io/en/v1.12.6/releases/1.12.html#improved-embed-configuration, this package will no longer be maintained.
